plugins/entando-plugin-jacms/src/test/java/com/agiletec/plugins/jacms/aps/system/services/resource/ResourceManagerIntegrationTest.java

```diff
@@ -388,7 +388,7 @@ public void testCreateResourceType() {
     public void testGetGroupUtilizers() throws Throwable {
         assertTrue(this.resourceManager instanceof GroupUtilizer);
         List utilizers = ((GroupUtilizer) this.resourceManager).getGroupUtilizers(Group.FREE_GROUP_NAME);
-        assertEquals(3, utilizers.size());
+        assertEquals(4, utilizers.size());
```

Comment:
What happened that makes this number change in this PR?

Reply:
I added a new resource in the script for test